prisma/app-ppg/src/realtime.ts

```diff
@@ -0,0 +1,41 @@
+import { PrismaClient } from '@prisma/client';
+import { withPulse } from '@prisma/extension-pulse';
+import dotenv from "dotenv"
+
+process.on('SIGINT', () => {
+  process.exit(0);
+});
+
+dotenv.config();
+const apiKey: string = process.env.PULSE_API_KEY ?? '';
+if (!apiKey || apiKey === "") {
+  console.log(`Please set the \`PULSE_API_KEY\` environment variable in the \`.env\` file.`);
```

Comment:
Would this not be setting in dev.nix?

Reply:
Ah good catch, thanks!

```suggestion
  console.log(`Please set the \`PULSE_API_KEY\` environment variable in the \`.idx/dev.nix\` file.`);
```